mint.go

```diff
@@ -22,14 +24,16 @@ func initJSONStore() error {
 	// Create the application
 	var app types.Application
 
-	session, err := mgo.Dial("localhost")
+	fmt.Println("running aginst mongo: ", os.Getenv("MONGO_URL"))
```

Comment:
If `Mongo_URL` env is not set, what does it use by default?

Reply:
it uses localhost:27017 but tbh its not obvious from the client library code... When I added it and  run it just still worked which surprised me too. (unless there is another bug I missed ;) )

Reply:
so turned out i was wrong and it doesn't default to localhost:27017.. must have forgotten i've added it. This way or the other im removing it as well as removing override for the ABCI port. Since you may be merging it there is no point in me forcing code basically added for testing